CHANGELOG.md

```diff
@@ -209,6 +209,8 @@ Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/
 
 The following are deprecated and will be removed in a future release. For anything deprecated in 4.26 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
 
+- For local builds, [Webpack](https://webpack.js.org/) versions prior to `5.84.0` are deprecated at 4.27. This is related to bug fixes in Webpack.
```

Comment:
shouldn't this be in the list with the other deprecations?

Reply:
I did think about it, I could put it at the top of the list? it didn't neatly fit in because it's not a class, property or method.

Reply:
I think it's fine here - the list doesn't specify it's only for classes/properties/events. We might have to do something different for the release notes though

Reply:
SGTM, it's okay for it to get a little more visibility here, rather than being buried in the list, in case anyone has feedback or questions.